bokehjs/src/coffee/core/has_props.coffee

```diff
@@ -373,23 +373,19 @@ class HasProps extends Backbone.Model
         throw new Error("models must be owned by only a single document")
       else
         @document._notify_attach(@)
```

Comment:
@bryevdv, `_notify_attach()` increments a reference counter, so it has to be called on every attach, even if document is attached to a given object. Given the current design, this is necessary, because otherwise when detaching a document, models would be detached too early (e.g. back references from tools or guides to plot). Originally I just added a return statement, but then tests failed and had to add this line.